password_generator/passgen.py

```
@@ -0,0 +1,55 @@
+parser.add_argument("minlen", type=int, nargs="?", default=8, help="Minimum length of the password (default: 8).")
+parser.add_argument("maxlen", type=int, nargs="?", default=12, help="Maximum length of the password (default: 12).")
+parser.add_argument("--lowercase", action="store_true", help="Include lowercase letters.")
+parser.add_argument("--uppercase", action="store_true", help="Include uppercase letters.")
+parser.add_argument("--special", action="store_true", help="Include special characters.")
+parser.add_argument("--numbers", action="store_true", help="Include numbers.")
+
+args = parser.parse_args()
+
+# Validate minlen and maxlen
+if args.minlen > args.maxlen:
+    raise ValueError("Minimum length cannot be greater than maximum length.")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)